index.html

```diff
@@ -1,32 +1,30 @@
-<!doctype html>
-<html>
-  <head>
-    <title>ATM</title>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="atm.css">
-  </head>
-  <body>
-    <div class="header">
-      <div><img src="ga.png" /></div>
-      <div class="title">Bank of GA</div>
-    </div>
-    
-    <div class="account">
-      <h2>Checking</h2>
-      <div class="balance">$0</div>
-      <input type="text" placeholder="enter an amount" />
-      <input type="button" value="Deposit" />
-      <input type="button" value="Withdraw" />
-    </div>
+<!DOCTYPE html>
+<title>ATM</title>
+<meta charset="utf8">
+<link rel="stylesheet" href="atm.css">
+<script src="https://code.jquery.com/jquery2.2.1.min.js"></script>
```

Comment:
script tag link is giving me an error, try this cdn:  

`<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>`